packages/database/package.json

```diff
@@ -3,23 +3,53 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
-  "type": "module",
   "exports": {
-    "./lib/*": "./src/lib/*.ts",
-    "./dbDotEnv": "./src/dbDotEnv.ts",
-    "./dbTypes": "./src/dbTypes.ts",
-    "./inputTypes": "./src/inputTypes.ts"
+    "./lib/*": {
+      "types": "./dist/src/lib/*.d.ts",
+      "require": "./dist/src/lib/*.js",
+      "default": "./dist/src/lib/*.js"
+    },
+    "./dbDotEnv": {
+      "types": "./dist/src/dbDotEnv.d.ts",
+      "require": "./dist/src/dbDotEnv.js",
+      "default": "./dist/src/dbDotEnv.js"
+    },
+    "./dbTypes": {
+      "types": "./dist/src/dbTypes.d.ts",
+      "require": "./dist/src/dbTypes.js",
+      "default": "./dist/src/dbTypes.js"
+    },
+    "./inputTypes": {
+      "types": "./dist/src/inputTypes.d.ts",
+      "default": "./dist/src/inputTypes.d.ts"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "lib/*": [
+        "dist/src/lib/*.d.ts"
+      ],
+      "./dbDotEnv": [
+        "dist/src/dbDotEnv.d.ts"
+      ],
+      "./dbTypes": [
+        "dist/src/dbTypes.d.ts"
+      ],
+      "./inputTypes": [
+        "dist/src/inputTypes.d.ts"
+      ]
+    }
   },
   "scripts": {
     "init": "supabase login",
-    "dev": "supabase start && tsx scripts/createEnv.ts && supabase functions serve",
+    "dev": "supabase start && tsx scripts/createEnv.mts && supabase functions serve",
     "stop": "supabase stop",
-    "check-types": "tsc --noEmit --skipLibCheck",
+    "check-types": "tsc --emitDeclarationOnly --skipLibCheck",
```

Comment:
Why do we want the check-types to emit declarations?  Aren't we just running a check here?

Reply:
If the package declarations are not generated, the check of the apps, that refer to those declarations, fails.

Reply:
@maparent 
Can you show me an example of what you are referring to?  This wasn't the case previously when I initially checked in `check-types`, and I just just changed them all to `noEmit` and it ran successfully for me

<img width="2225" height="2042" alt="image" src="https://github.com/user-attachments/assets/39ab04dc-0b72-426f-b112-b4f86d7657a7" />

I also cd packages/database and ran `npm run check-types` successfully
<img width="1188" height="164" alt="image" src="https://github.com/user-attachments/assets/a2aec92c-89cd-48ea-8ce9-0d24ea524694" />